src/libraries/System.Numerics.Tensors/src/System/NRange.cs

```diff
@@ -89,21 +89,21 @@ public override string ToString()
             return new string(span.Slice(0, pos));
         }
 
-        /// <summary>Create an NRange object starting from start NIndex to the end of the collection.</summary>
+        /// <summary>Creates an NRange object starting from start NIndex to the end of the collection.</summary>
         public static NRange StartAt(NIndex start) => new NRange(start, NIndex.End);
 
-        /// <summary>Create an NRange object starting from first element in the collection to the end NIndex.</summary>
+        /// <summary>Creates an NRange object starting from first element in the collection to the end NIndex.</summary>
         public static NRange EndAt(NIndex end) => new NRange(NIndex.Start, end);
 
-        /// <summary>Create an NRange object starting from first element to the end.</summary>
+        /// <summary>Creates an NRange object starting from first element to the end.</summary>
         public static NRange All => new NRange(NIndex.Start, NIndex.End);
 
-        /// <summary>Calculate the start offset and length of NRange object using a collection length.</summary>
-        /// <param name="length">The length of the collection that the NRange will be used with. length has to be a positive value.</param>
+        /// <summary>Calculates the start offset and length of the NRange object using a collection length.</summary>
+        /// <param name="length">The length of the collection that the NRange will be used with. Must be a positive value.</param>
```

Comment:
```suggestion
        /// <summary>Calculates the start offset and length of the <see cref="NRange"/> object using a collection length.</summary>
        /// <param name="length">The length of the collection that the <see cref="NRange"/> will be used with. Must be a positive value.</param>
```

Reply:
Wow, how do you comment on multiple lines?